homeassistant/components/synology_dsm/const.py

```diff
@@ -247,5 +249,27 @@
     },
 }
 
+INFORMATION_SENSORS = {
+    f"{SynoDSMInformation.API_KEY}:temperature": {
+        ENTITY_NAME: "temperature",
+        ENTITY_UNIT: None,
+        ENTITY_ICON: None,
+        ENTITY_CLASS: "temperature",
```

Comment:
Please import and use the relavant device class constants from the sensor integration.

https://github.com/home-assistant/core/blob/f187091594b08e64e5c34bb84b729fd01f01f417/homeassistant/components/sensor/__init__.py#L9-L20